saleor/payment/gateways/np_atobarai/tests/test_plugin.py

```diff
@@ -1,7 +1,12 @@
+import logging
 import os
-from unittest.mock import Mock, patch
+from unittest.mock import ANY, Mock, patch
 
+import pytest
 import requests
+from graphene import Node
+
+from saleor.payment import PaymentError
```

Comment:
Would be good to have unified way of imports between files. Looks like in the rest of the test files you're using relative.